### PR TITLE
Migrate to Flux v2 AKS Extension

### PR DESCRIPTION
**Pull request by ckittel**
* Move from bootstrapping the cluster with Flux manually after deployment to using the Flux AKS Extension to bootstrap the cluster WITH the deployment.

* Update to AKS 1.22.4

* Update to Kured 1.9.0

### CODE REVIEW

File: 05-bootstrap-prep.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Prep for cluster bootstrapping
+
+Now that the [hub-spoke network is provisioned](./04-networking.md), the next step in the [AKS secure Baseline reference implementation](./) is preparing what your AKS cluster should be bootstrapped with.
+
+## Expected results
+
+Container registries often have a lifecycle that extends beyond the scope of a single cluster. They can be scoped broadly at organizational or business unit levels, or can be scoped at workload levels, but usually are not directly tied to the lifecycle of any specific cluster instance. For example, you may do blue/green _cluster instance_ deployments, both using the same container registry. Even though clusters came and went, the registry stays in tact.
```
Comment (magrande) [inline 05-bootstrap-prep.md:7]: is "In tact" ok here ?  Or the adjective "intact" is better usage ?  I'm not sure, but just in case

Comment (ckittel) [reply 05-bootstrap-prep.md:7]: Good catch.

**Review by magrande — APPROVED** [review]
LGTM !

File: 05-bootstrap-prep.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+* Azure Container Registry (ACR) is deployed, and exposed as a private endpoint.
+* ACR is populated with images your cluster will need as part of its bootstrapping process
+* Log Analytics is deployed and ACR platform logging is configured.  This workspace will be used by your cluster as well.
```
[the remark this answers is not in the document]
Comment (ckittel) [reply 05-bootstrap-prep.md]: Addressed in https://github.com/mspnp/aks-baseline/pull/256/commits/ffd152968c9a8e163aa8707015a4fcc895447dcf

**Comment by ckittel** [comment]
Thanks for your review @magrande

**Pull request merged**